subtitles/subtitles.py

```diff
@@ -77,56 +76,63 @@ def __generate(self, transcriber: VoskTranscriber, source: str, stream_id: str,
                 logging.error(err)
 
 
-def serve(properties: dict, debug: bool = False) -> None:
+def serve(transcriber: Transcriber,
+          receiver: str,
+          port: int,
+          debug: bool = False) -> None:
     """Starts the grpc server.
 
     Args:
-        properties: The configuration of the server.
+        transcriber: The transcriber used.
+        receiver: The network address of the receiver.
+        port: The port on which the voice service listens.
         debug: Whether the server should be started in debug mode or not.
     """
-    transcriber = get_transcriber(properties)
-    receiver = f'{properties["receiver"]["host"]}:{properties["receiver"]["port"]}'
 
-    server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))  # TODO: How to determine how many workers? Guess?
-    subtitles_pb2_grpc.add_SubtitleGeneratorServicer_to_server(
-        servicer=SubtitleServerService(transcriber, receiver),
-        server=server)
+    with ThreadPoolExecutor(max_workers=None) as executor:
```

Comment:
Just a last suggestion:
Do we want to make `max_workers` configurable via environment-variables/ the yaml file?

Reply:
Sure. Great idea!